lib/renderer/ipc-renderer-internal-utils.ts

```diff
@@ -1,14 +1,14 @@
-'use strict'
-
-const ipcRenderer = require('@electron/internal/renderer/ipc-renderer-internal')
-const errorUtils = require('@electron/internal/common/error-utils')
+import * as ipcRenderer from '@electron/internal/renderer/ipc-renderer-internal'
+import * as errorUtils from '@electron/internal/common/error-utils'
 
 let nextId = 0
 
-exports.invoke = function (command, ...args) {
+export function invoke (command: string, ...args: any[]) {
   return new Promise((resolve, reject) => {
```

Comment:
Can this be `T`?


```js
export function invoke<T=any>(command: string, ...args: any[]) {
  return new Promise<T>((resolve, reject) => {
```

The default resolve for `Promise` is `{}` which isn't very helpful